### PR TITLE
MAUI-900342-Prepare Vol 3 Calendar Feature

### PR DESCRIPTION
**Pull request by Sathish-087**
**Description**
Added calendar header and view header template support UG.

### CODE REVIEW

File: MAUI/Calendar/headers.md [hunk 1]
```diff
@@ -142,4 +241,76 @@ this.Calendar.MonthView.HeaderView = new CalendarMonthHeaderView
...
+                    <Label x:Name="label" HorizontalOptions="Center" VerticalOptions="Center" Text="{Binding StringFormat='{0:ddd}'}" TextColor="White" FontSize="14" FontFamily="Bold" />
+                </Grid>
+            </DataTemplate>
+        </Calendar:SfCalendar.MonthViewHeaderTemplate>
+ </Calendar:SfCalendar>
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view View Header appearance changed using template in .NET MAUI Calendar.](images/header/maui-month-view-view-header-data-template.png)
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/headers.md:267]: get proper images from UX team

File: MAUI/Calendar/headers.md [hunk 2]
```diff
@@ -72,6 +72,105 @@ this.Calendar.HeaderView = new CalendarHeaderView()
...
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view Header appearance changed using template .NET MAUI Calendar.](images/header/maui-month-view-header-data-template.png)
+
+## Header appearance using DataTemplateSelector
+
+You can customize the header appearance by using the [HeaderTemplate]() in the [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html). The `DataTemplateSelector` can choose a `DataTemplate` at runtime based on the value of a data-bound to Calendar header by using the `HeaderTemplate.` It allows you to choose a different data template for each header, as well as to customize the appearance of a particular header based on certain conditions.
+
```
Comment (MuniappanSubramanian) [inline MAUI/Calendar/headers.md:109]: You can customize the header appearance by using the [HeaderTemplate](#) in the [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html). The `DataTemplateSelector` allows you to choose a `DataTemplate` at runtime based on the value bound to the calendar header. This lets you select a different data template for each header and customize the appearance of a specific header based on certain conditions.

Like this prepare view header

File: MAUI/Calendar/headers.md (hunk 1 above)
Comment (Sathish-087) [reply MAUI/Calendar/headers.md:270]: Image changed.

File: MAUI/Calendar/headers.md (hunk 2 above)
Comment (Sathish-087) [reply MAUI/Calendar/headers.md:109]: Content changed.

File: MAUI/Calendar/headers.md [hunk 3]
```diff
@@ -72,6 +72,105 @@ this.Calendar.HeaderView = new CalendarHeaderView()
...
+        {
+            if (headerDetails.StartDateRange.Date <= DateTime.Now.Date && headerDetails.EndDateRange >= DateTime.Now.Date)
+                return TodayDatesTemplate;
+        }
+        return NormaldatesTemplate;
+    }
+}
+
+{% endhighlight %}  
+{% endtabs %}
```
[the remark this answers is not in the document]
Comment (Sathish-087) [reply MAUI/Calendar/headers.md:172]: Notes added.

**Review by MuniappanSubramanian — APPROVED** [review]

**Pull request merged**